tests/backup/backup_locked_bucket_test.go

```diff
@@ -412,3 +412,236 @@ var _ = Describe("{LockedBucketResizeOnRestoredVolume}", func() {
+								params["auth-token"], err = Inst().S.GetTokenFromConfigMap(Inst().ConfigMap)
+								dash.VerifyFatal(err, nil, "Fetching token from configMap")
+							}
+							err := Inst().V.ValidateUpdateVolume(volume, params)
+							dash.VerifyFatal(err, nil, "Validate volume update status")
+						}
+					}
+				})
+				Step("Checking size of volume after resize", func() {
+					log.InfoD("Checking size of volume after resize")
+					pods, err := core.Instance().GetPods(namespace, labelSelectors)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the pod list"))
+					srcClusterConfigPath, err := GetSourceClusterConfigPath()
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Getting kubeconfig path for source cluster %v", srcClusterConfigPath))
+					for _, pod := range pods.Items {
+						afterSize, err := getSizeOfMountPoint(pod.GetName(), namespace, srcClusterConfigPath)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the mount size %v", afterSize))
+						podListAfterSizeMap[pod.Name] = afterSize
+					}
+					for _, pod := range pods.Items {
+						dash.VerifyFatal(podListAfterSizeMap[pod.Name] > podListBeforeSizeMap[pod.Name], true, fmt.Sprintf("Verifying volume size for pod %s", pod.Name))
+					}
+				})
+				Step("Create schedule backup after initializing volume resize", func() {
+					log.InfoD("Create schedule backup after initializing volume resize")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					log.FailOnError(err, "Unable to px-central-admin ctx")
+					preRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, preRuleNameList[i])
+					postRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, postRuleNameList[i])
+					scheduleName = fmt.Sprintf("%s-schedule-%v", BackupNamePrefix, time.Now().Unix())
+					labelSelectors := make(map[string]string)
+					err = CreateScheduleBackup(scheduleName, SourceClusterName, backupLocationName, backupLocationUID, []string{namespace},
+						labelSelectors, orgID, preRuleNameList[i], preRuleUid, postRuleNameList[i], postRuleUid, periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule name [%s]", scheduleName))
+					firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+				})
+				Step("Verifying backup success after initializing volume resize", func() {
+					log.InfoD("Verifying backup success after initializing volume resize")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					log.FailOnError(err, "Unable to px-central-admin ctx")
+					firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+					err = backupSuccessCheck(firstScheduleBackupName, orgID, 0, 0, ctx)
```

Comment:
we need to pass the required timeout and retry value here, backupSuccessCheck wont initalize any default value's